users/forms.py

```diff
@@ -18,6 +20,28 @@ class Meta:
         fields = ("email",)
 
 
+class SimpleUserChangeForm(ModelForm):
+    """
+    A form for updating users by EMPLOYEE users..
```

Comment:
One unnecessary dot at the end ;)

Reply:
About the comment itself, I think it is not possible to define how this form will be used in the future, so I would drop `by EMPLOYEE users` part. As we all know those comments get outdated very quickly ;)